scripts/local/run.sh

```diff
@@ -17,9 +17,10 @@ function printHelp {
 }
 
 function cleanup {
-    echo "Shutting down network before exiting..."
-    ./scripts/local/run_stop.sh
-    echo "Network stopped"
+    echo "Shutting down and cleaning network before exiting..."
+    rm -f NETWORK_READY
+    docker compose -f docker/docker-compose-run.yml --project-directory ./ stop
+    echo "Network cleaned and stopped"
```

Comment:
The `trap` in `run_setup.sh` only calls `avalanche network stop`, so I think we should leave the `echo`s as is.

I could see a scenario in which someone stops the network by ctrl-c'ing out of the `run.sh` process, and then want to start the stopped containers to perform further manual testing.

We should document that subsequent calls of `run.sh` _will_ clean the network, but we should make it clear in our `echo`s exactly when that cleaning takes place, as well as document it in the README.